### ISSUE TITLE
Is DragonBones 4.0 works with Unity?

### ISSUE THREAD

**Issue by NDTnarathorn** [DragonBones/DragonBonesUnity#8]
Is DragonBones 4.0 works with Unity?

**Comment by BenoitFreslon** [comment]
+1

**Comment by maxin0628** [comment]
No, as far as I know nobody except me is working on that, but about 4.0, I've just translated the code, but no further effort on that because I've been not making gaming since mid of this year...
